### DISCUSSION TITLE
Markdown Editor

### DISCUSSION THREAD

**Discussion by mattlevine32** [remirror/remirror#693] [General]
Hey, really love this project. Was wondering when you're planning on adding the markdown editor mentioned in the docs as coming soon?

**Comment by ifiokjr** [comment]
Thanks for the kind words. 

Markdown support is a goal and it's in the [backlog](https://github.com/remirror/remirror/issues/553) right now. You can upvote [the issue](https://github.com/remirror/remirror/issues/553) to show your support.
